public/stylesheets/site/2.0/26-media-narrow.css

```diff
@@ -152,10 +159,11 @@ body .narrow-shown {
   padding: 0;
 }
 
-.splash div.module, .logged-in .splash > .module {
-  margin-left: 1%;
-  margin-right: 1%;
-  width: 98%;
+.splash div.module, .logged-in .splash div.module {
```

Comment:
Do not want 4.5% margins on the homepage. That'd be weird when they're 3.5% everywhere else.